src/OSPSuite.Core/Domain/Services/SensitivityAnalyses/ISensitivityAnalysisTask.cs

```diff
@@ -11,24 +11,29 @@ public interface ISensitivityAnalysisTask
       SensitivityAnalysis CreateSensitivityAnalysis();
 
       void AddToProject(SensitivityAnalysis sensitivityAnalysis);
-      
+
       /// <summary>
-      /// Asks the user if he really wants to delete the <paramref name="sensitivityAnalyses"/>. Returns <c>true</c> if the <paramref name="sensitivityAnalyses"/>
-      /// are deleted from the current project otherwise <c>false</c>
+      ///    Asks the user if he really wants to delete the <paramref name="sensitivityAnalyses" />. Returns <c>true</c> if the
+      ///    <paramref name="sensitivityAnalyses" />
+      ///    are deleted from the current project otherwise <c>false</c>
       /// </summary>
       bool Delete(IReadOnlyList<SensitivityAnalysis> sensitivityAnalyses);
 
       void SwapSimulations(SensitivityAnalysis sensitivityAnalysis, ISimulation oldSimulation, ISimulation newSimulation);
-      bool ValidateSwap(SensitivityAnalysis sensitivityAnalysis, ISimulation oldSimulation, ISimulation newSimulation);
```

Comment:
This task was using some presenter stuff. I had to remove it outside of usage so that I can reuse. This is better now